### ISSUE TITLE
weight decay?

### ISSUE THREAD

**Issue by mdering** [MarcBS/keras#21]
hi Marc,
I'm having trouble figuring out if your code takes into account the weight decay (or decay_mult) parameter of caffe models. That should be L2 weight regularizer right?
